__tests__/app.tests.js

```diff
@@ -68,13 +67,103 @@ describe("GET api/articles/ID ", () => {
         expect(output).toHaveProperty("article_img_url");
       });
   });
+  test("GET articles by id should return article with properties", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((res) => {
+        const output = res.body.articles[0];
+        expect(output.author).toEqual(expect.any(String));
+        expect(output.title).toEqual(expect.any(String));
+        expect(output.body).toEqual(expect.any(String));
+        expect(output.topic).toEqual(expect.any(String));
+        expect(output.created_at).toEqual(expect.any(String));
+        expect(output.article_img_url).toEqual(expect.any(String));
+        expect(output.article_id).toEqual(expect.any(Number));
+        expect(output.votes).toEqual(expect.any(Number));
+      });
+  });
   test("GET articles by id should return a 404 and msg of 'no such article !' if id is not in the db", () => {
     return request(app)
       .get("/api/articles/90")
       .expect(404)
       .then((res) => {
-        console.log(res.body.msg);
         expect(res.body.msg).toEqual("No such article !");
       });
   });
 });
+describe("GET api/article", () => {
+  test("GET API/article should return an array of article objects ", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((res) => {
+        const output = res.body.articles[0];
+        expect(typeof res.body).toEqual("object");
+        expect(output).toHaveProperty("author");
+        expect(output).toHaveProperty("title");
+        expect(output).toHaveProperty("article_id");
+        expect(output).toHaveProperty("topic");
+        expect(output).toHaveProperty("created_at");
+        expect(output).toHaveProperty("votes");
+        expect(output).toHaveProperty("article_img_url");
+      });
+  });
+  test("GET articles with correct properties", () => {
```

Comment:
Is this a test for the articles/:id endpoint? We seem to be mixing tests up for this endpoint and just the /articles endpoint.

General testing comments as discussed in our call:
Object rather than array for single item response
use a forEach when we have arrays of items to check each index
We don't need to check for a key if we're asserting against the type of the key value
Check ids if using parametric endpoints for these.